README.md

```diff
@@ -132,13 +132,13 @@ To configure socket channel provider you should implements SocketChannelProvider
 For example:
 
 ```
-tarantool://localhost:3301?user=test&password=test&socketProvider=abc.xyz.MySocketProvider
+jdbc:tarantool://localhost:3301?user=test&password=test&socketProvider=abc.xyz.MySocketProvider
```

Comment:
It's required by the standard (spec, paragraph 9.4):
> The format of a JDBC URL is :
> - jdbc: _subprotocol_ : _subname_

I also checked some implementations and all of them follow the format without an exception:

```text
jdbc:ignite:thin://host:port?params
jdbc:postgresql://host:port/database
jdbc:mysql://host:port/database
jdbc:mariadb://host:port/database
jdbc:oracle:driver_type:[username/password]@database_specifier
jdbc:sqlserver://host:port
jdbc:h2:tcp://host:port/~/database
jdbc:hsqldb:http://host:port/database
jdbc:derby:database
```

Reply:
It worth to explicitly mention the incompatible change in the commit message. At least it will decrease probability to miss this when we'll write release notes.